test/alternator/test_describe_table.py

```diff
@@ -18,7 +18,15 @@
 from botocore.exceptions import ClientError
 import re
 import time, datetime
-from test.alternator.util import multiset, create_test_table, new_test_table
+from test.alternator.util import multiset, create_test_table, new_test_table, random_string
+from test.cqlpy.util import config_value_context
```

Comment:
I'm surprised this even works :-) I guess it does, because it is possible to use CQL from Alternator tests, but it's usually not a good idea. We use CQL for rare things like in test_cql_rbac.py, but should avoid it in Alternator tests as much as we can...

The real way to do this temporary configuration is with https://github.com/scylladb/scylladb/pull/19147. I'll need to see why it never got in :-(